docs/runbooks/KubevirtVmHighMemoryUsage.md

```diff
@@ -23,7 +23,7 @@ $ kubectl exec -it <virt-launcher> -c compute -- top
 
 ## Mitigation
 
-Increase the memory limit in the `VirtualMachine` specification as in the following example:
+Increase the memory limit in the `VirtualMachine` specification:
```

Comment:
I think we should mention its an example. No?

Reply:
Or do like for the docs/runbooks/KubeVirtComponentExceedsRequestedMemory.md Where the values are stated like `<memory_limit>` `<memory_request>`

Reply:
The example in docs/runbooks/KubeVirtComponentExceedsRequestedMemory.md is for upstream only. Would the downstream example look the same?